CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.2.1] - 2023-02-22
```

Comment:
Remove this, it is not needed, it'll be set automatically on release

Reply:
Version removed